src/components/GradeItemAccordion.tsx

```diff
@@ -0,0 +1,220 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import TextInput from './TextInput';
+import DelButton from './MainActionButton';
+import { CourseItem, Fraction } from '../api';
+export interface GradeItemAccordionInterface {
+	item: CourseItem;
+	updateItem: (item: CourseItem) => void;
+}
+
+const Accordion = styled.tr`
+	background-color: #ffffff;
+	color: #4f4f4f;
+	padding: 2em;
+	justify-content: center;
+	line-height: 8em;
```

Comment:
I am kind of confused what this property does, but I think it may get changed during the styling ticket anyways.